pom.xml

```diff
@@ -81,7 +81,7 @@
     <test_source_directory>${genomicsdb_source_directory}/src/test/java</test_source_directory>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <surefireArgLine>-Djava.library.path=${project.basedir}/bin:${java.library.path}</surefireArgLine>
+    <surefireArgLine>--illegal-access=permit -Djava.library.path=${project.basedir}/bin:${java.library.path}</surefireArgLine>
```

Comment:
Why have `--illegal-access=permit`? Is this causing errors or warnings? And should we allow this at all?

Reply:
Previously that property was being set if it was a Java 11 build. Java 17 doesn't support that command at all, so it was being ignored, that's why I put it there. I think that was put for java 11 because Hadoop was generating `Illegal reflective access`. Since I upgraded to newer Hadoop version, Hadoop no longer generates that error. It should be safe to remove it.